tests/test_io/test_io_flip.py

```diff
@@ -0,0 +1,12 @@
+import magma as m
```

Comment:
let's just name this file `test_io.py` since we don't have a basic test for `IO()` class. Eventually we can add more testing for the IO and derived classes. (not really sure what the other test files in this directory are doing, but probably can clean them up and move them later).